consensus/rpcclient/client.go

```diff
@@ -0,0 +1,100 @@
+package consensusrpcclient
+
+import (
+	"context"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/node"
+
+	"github.com/offchainlabs/nitro/arbos/arbostypes"
+	"github.com/offchainlabs/nitro/arbutil"
+	"github.com/offchainlabs/nitro/consensus"
+	"github.com/offchainlabs/nitro/execution"
+	"github.com/offchainlabs/nitro/util/containers"
+	"github.com/offchainlabs/nitro/util/rpcclient"
+	"github.com/offchainlabs/nitro/util/stopwaiter"
+)
+
+type ConsensusRPCClient struct {
+	stopwaiter.StopWaiter
+	client *rpcclient.RpcClient
+}
+
+func NewConsensusRpcClient(config rpcclient.ClientConfigFetcher, stack *node.Node) *ConsensusRPCClient {
```

Comment:
nitpick: golang naming convention

```suggestion
func NewConsensusRPCClient(config rpcclient.ClientConfigFetcher, stack *node.Node) *ConsensusRPCClient {
```